src/framework/components/app/core.clj

```diff
@@ -3,7 +3,7 @@
     [cats.core :as m]
     [com.stuartsierra.component :as component]
     [reitit.core :as r]
-    [xiana.commons :refer :all]
+    [xiana.commons :refer [?assoc-in]]
```

Comment:
I intended xiana.commons as a place for our clojure.core extensions, so this was intent behind for `:refer :all` which is usually to be avoided